data/json/monster_factions.json

```diff
@@ -217,6 +217,11 @@
     "name": "corvid",
     "base_faction": "mutant"
   },
+  {
+    "type": "MONSTER_FACTION",
+    "name": "lab_mutant",
+    "base_faction": "mutant"
```

Comment:
So just the default for mutant vs science, bots, etc?

Reply:
Yes, they are escaped lab subjects (either from transports-to-come or through the tunnels after breaking out) so they shouldn't be friendly to security. I like them in packs, and they have some potential to flesh out - I'm thinking something along the line of DF's animal people, variants with different mutations, keeping creepy crawly subway critters as pets, rudimentary camps, that sort of thing.

Reply:
How about the other way around - HATE vs defense_bot?